test/test-basic-test.jl

```diff
@@ -1,3 +1,28 @@
+using InclusiveScans
+using CUDA
+
 @testset "InclusiveScans.jl" begin
```

Comment:
I suggest testing this for random `BitVectors` because this is our actual use case.